### PR TITLE
Update README.md lala

### PR DESCRIPTION
**Pull request by viktoriia-bilous**

### CODE REVIEW

**Comment by kudrykv** [comment]
my comment

**Comment by kudrykv** [comment]
hello world

**Comment by kudrykv** [comment]
foo bar baz

**Comment by kudrykv** [comment]
quux

File: README.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # check-notifications
 
-fancy a pr
+🫶🙆‍♀️🐛
```
Comment (kudrykv) [inline README.md:3]: 🤪

**Review by kudrykv — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (viktoriia-bilous) [reply README.md:3]: 🦞 👬 🦞

**Pull request closed without merging**